apps/homepage/src/app/[lang]/_shared/header/header.tsx

```diff
@@ -1,6 +1,6 @@
 "use client";
 import styles from "./header.module.scss";
-import { MenuButton } from "../menuButton";
+import { MenuButton } from "@1xdev/ui";
```

Comment:
I would suggest a different naming, something like `IconLink` think about what this does, not about how you will use it as we might use it in a different context now that is in a shared package that is published publicly.